cluster/gce/manifests/cluster-autoscaler.manifest

```diff
@@ -17,7 +17,7 @@
         "containers": [
             {
                 "name": "cluster-autoscaler",
-                "image": "k8s.gcr.io/cluster-autoscaler:v1.16.0-beta.1",
+                "image": "k8s.gcr.io/cluster-autoscaler:v1.16.0",
```

Comment:
what level of kube libraries is this built against?

Reply:
We are vendoring huge part of k8s (both internals and clients taken directly from staging). This build was vendored agains k8s tagged as v1.16.0-beta.2

Reply:
I did not revendor before this build so delta is as minimal as possible (literally just file containing version number changed).